static/js/directives/ui/quota-management-view.js

```diff
@@ -20,51 +20,67 @@ angular.module('quay').directive('quotaManagementView', function () {
       $scope.rejectLimitType = 'Reject';
+      $scope.errorMessage = '';
+      $scope.errorMessagesObj = {
+        'quotaLessThanZero': 'A quota greater 0 must be defined.',
+        'quotaLimitNotInRange': 'A quota limit greater 0 and less than 100 must be defined.',
+        'validNumber': 'Please enter a valid number.',
+        'setQuotaBeforeLimit': 'Please set quota before adding a quota Limit.',
+        'singleRejectLimit': 'Error: A quota policy should only have a single reject threshold.',
+        'identicalThresholds': 'Error: The quota policy contains two identical thresholds.',
+      };
+      $scope.warningMessage = '';
+      $scope.warningMessagesObj = {
+        'noQuotaLimit': 'Note: No quota policy defined. Users will be able to exceed the storage quota set above.',
+        'noRejectLimit': 'Note: This quota has no hard limit enforced via a rejection thresholds. Users will be able to exceed the storage quota set above.',
+      }
+      $scope.showConfigPanel = false;
 
       var loadOrgQuota = function () {
         $scope.nameSpaceResource = ApiService.listOrganizationQuota(
           null, {'orgname': $scope.organization.name}
         ).then((resp) => {
-            if (resp.length > 0) {
-              quota = resp[0];
-              $scope.prevQuotaConfig['id'] = quota["id"];
-	      $scope.currentQuotaConfig['id'] = quota["id"];
-              
-              for (i in quota['limits']) {
-                limitId = quota['limits'][i]['id'];
-                $scope.prevQuotaConfig['limits'][limitId] = $.extend({}, quota["limits"][i]);
-		$scope.currentQuotaConfig['limits'][limitId] = $.extend({}, quota["limits"][i]);
-              }
-
-              let { result, byte_unit } = normalizeLimitBytes(quota["limit_bytes"]);
-              $scope.prevQuotaConfig['quota'] = result;
-	      $scope.currentQuotaConfig['quota'] = result;
-              $scope.prevQuotaConfig['byte_unit'] = byte_unit;
-	      $scope.currentQuotaConfig['byte_unit'] = byte_unit;
-
-              if (quota["limit_bytes"] != null) {
-                $scope.prevquotaEnabled = true;
-              }
-
-	      $scope.organization.quota_report.configured_quota = quota["limit_bytes"];
-	      $scope.organization.quota_report.percent_consumed = (parseInt($scope.organization.quota_report.quota_bytes) / $scope.organization.quota_report.configured_quota * 100).toFixed(2);
+          if (resp.length > 0) {
+            const quota = resp[0];
+            $scope.prevQuotaConfig['id'] = quota["id"];
+            $scope.currentQuotaConfig['id'] = quota["id"];
+
+            for (const i in quota['limits']) {
+              const limitId = quota['limits'][i]['id'];
+              $scope.prevQuotaConfig['limits'][limitId] = $.extend({}, quota["limits"][i]);
+              $scope.currentQuotaConfig['limits'][limitId] = $.extend({}, quota["limits"][i]);
             }
+
+            let {result, byte_unit} = normalizeLimitBytes(quota["limit_bytes"]);
```

Comment:
Why did this get moved outside the if block. quota will not be guarenteed to be not null here.

Reply:
It is inside the `resp.length > 0` if block. Similar to as it was before. Only changes here are indentations.